packages/runtime/src/useCases/transport/identityRecoveryKits/CreateIdentityRecoveryKit.ts

```diff
@@ -43,11 +51,14 @@ export class CreateIdentityRecoveryKitUseCase extends UseCase<CreateIdentityReco
 
         const newBackupDevice = await this.devicesController.sendDevice({ isAdmin: true, isBackupDevice: true, name: "Backup Device" });
         const sharedSecret = await this.devicesController.getSharedSecret(newBackupDevice.id, request.profileName);
+
+        const passwordProtection = { ...request.passwordProtection, passwordLocationIndicator: "RecoveryKit" as PasswordLocationIndicator };
```

Comment:
Why do you have to cast?

Reply:
Because below PasswordProtectionCreationParameters.create expects the type PasswordLocationIndicator and not string.

Reply:
Well, apparently the compiler needs it more explicitly anyhow.

Reply:
Hm, why don't you use 
```suggestion
        const passwordProtection = { ...request.passwordProtection, passwordLocationIndicator: PasswordLocationIndicator.RecoveryKit };
```
?

Reply:
Because PasswordLocationIndicator refers to a type. Maybe you mean PasswordLocationIndicatorOptions.RecoveryKit. However, that is equal to 0 and thus not included in the type PasswordLocationIndicator, where the numbers only start at 50.

Reply:
eeeeeeeh